src/PlusDataCollection/Capistrano/vtkPlusCapistranoVideoSource.cxx

```diff
@@ -683,15 +685,15 @@ PlusStatus vtkPlusCapistranoVideoSource::WriteConfiguration(vtkXMLDataElement* r
 // ----------------------------------------------------------------------------
 PlusStatus vtkPlusCapistranoVideoSource::NotifyConfigured()
 {
-  if( this->OutputChannels.size() > 1 )
+  if (this->OutputChannels.size() > 1)
   {
     LOG_WARNING("vtkPlusIntersonVideoSource is expecting one output channel and there are "
-                 << this->OutputChannels.size() << " channels. First output channel will be used.");
+                << this->OutputChannels.size() << " channels. First output channel will be used.");
   }
 
-  if( this->OutputChannels.empty() )
+  if (this->OutputChannels.empty())
   {
-    LOG_ERROR("No output channels defined for vtkPlusIntersonVideoSource. Cannot proceed." );
+    LOG_ERROR("No output channels defined for vtkPlusIntersonVideoSource. Cannot proceed.");

```

Comment:
This should be vtkPlusCapistranoVideoSource. I guess Capistrano wrapper was based on Interson one :smile: